wof/core_1_1.py

```diff
@@ -589,7 +589,7 @@ def create_source_element(self, sourceResult):
 
         contactInfo = self.create_contact_info_element(sourceResult)
 
-        source.ContactInformation = contactInfo
+        source.contactInformation = [contactInfo]
```

Comment:
I am not sure how the API is supposed to work here, nor we have enough test coverage to deduce that from the tests. But maybe we should check if `contactInfo` is already a list and listify it only if not? To avoid making a list of list and breaking things downstream.